typescript/web/src/connectors/repository/image-processing.ts

```diff
@@ -10,6 +14,8 @@ const maxImageSizePixel: { [mimetype: string]: number } = {
   "image/png": 60e6,
 };
 
+type ThumbnailSizes = 20 | 50 | 100 | 200 | 500;
+
```

Comment:
This type is already declared in another module

Reply:
Done in 5be069f91fcdabd84671f60ec3842daa55b883da